app/scss/resultPage.scss

```diff
@@ -1,9 +1,13 @@
+@import "imports/variables";
+@import url('https://fonts.googleapis.com/css?family=Asap&display=swap');
+
 html {
   font-size: 62.5%;
```

Comment:
this font-size reset is in styles.scss (in branch s1t5).  I think it will be duplicated once the branches are merged into story1

Reply:
legacy code